.travis.yml

```diff
@@ -2,19 +2,9 @@ language: c
 sudo: false
 matrix:
   include:
-    - env: GHCVER=7.8.4 CABALVER=1.22 COVERAGE=true
-      compiler: ": #GHC 7.8.4 - tests"
-      # ^ HACK before https://github.com/travis-ci/travis-ci/issues/4393 is resolved
-      addons: {apt: {packages: [cabal-install-1.22,ghc-7.8.4,happy-1.19.5,alex-3.1.4], sources: [hvr-ghc]}}
-    - env: GHCVER=7.6.3 CABALVER=1.22
-      compiler: ": #GHC 7.6.3"
-      addons: {apt: {packages: [cabal-install-1.22,ghc-7.6.3,happy-1.19.5,alex-3.1.4], sources: [hvr-ghc]}}
     - env: GHCVER=7.10.1 CABALVER=1.22 DEPLOY=yes
```

Comment:
Is there any reason why we deploy the 7.10.1 version? Should we switch to the most recent version?